docs/src/dev-guide/components-webui.md

```
@@ -1,101 +1,70 @@
 
-We enforce code quality and consistency across our project using [ESLint][eslint]. Due to specific
-dependencies, linting this project requires Node.js v18 or higher. We offer two methods for
-performing linting; you may choose either one according to your preference.
+    ```shell
+    cd components/webui
+    (cd client && npm i)
+    (cd server && npm i)
+    ```
 
-### Method 1: Run `Taskfile` tasks
+    If you add a package manually to `package.json` or `package.json` changes for some other reason, 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)